src/components/List/List.tsx

```diff
@@ -261,18 +172,71 @@ export class List extends React.Component {
             }
         }
     };
-    onFocus = () => {
+
+    private getContainer() {
+        const ref = this.refContainer.current;
+        const wrappedInstance =
+            ref &&
+            'getWrappedInstance' in ref &&
+            typeof ref.getWrappedInstance === 'function' &&
+            ref.getWrappedInstance();
+
+        return this.props.sortable ? wrappedInstance : ref;
+    }
+
+    private filterItem = (filter: string) => (item: ListItemData<T>) => {
+        return String(item).includes(filter);
+    };
+
+    private getFilter() {
+        const {filter = this.state.filter} = this.props;
+        return filter;
+    }
+
+    private getItemsStyle() {
+        let {itemsHeight} = this.props;
+
+        if (typeof itemsHeight === 'function') {
+            itemsHeight = itemsHeight(this.state.items);
+        }
+
+        return itemsHeight ? {height: itemsHeight} : undefined;
+    }
+
+    private scrollToIndex = (index: number) => {
+        const container = this.getContainer();
+
+        if (container) {
+            container.scrollToItem(index);
+        }
+    };
+
+    private deactivate = () => {
+        if (this.props.deactivateOnLeave) {
+            this.setState({activeItem: undefined});
+        }
+    };
+
+    private handleKeyMove(event: React.KeyboardEvent, step: number, defaultItemIndex = 0) {
+        event.preventDefault();
+        const {activeItem = defaultItemIndex} = this.state;
+        this.activateItem(List.findNextIndex(this.state.items, activeItem + step, Math.sign(step)));
+    }
+
+    private onFocus = () => {
```

Comment:
lets name handler methods like this: `handle*`, `on*` functions are usually come from component props

Reply:
Ok. But someone is already using `onKeyDown` public method according to this [type](https://github.com/yandex-cloud/uikit/blob/main/src/components/List/List.d.ts#L65). I'll leave it as it is.

Reply:
Leave a comment to change it in the next major please